docs/REST API/rest-api.md

```diff
@@ -8918,6 +8987,7 @@ Deletes an existing script.
   {
     "script_id": 3,
     "name": "remove-zoom-artifacts.sh",
+    "hashed_contents": "<HASH>",
```

Comment:
@noahtalerman do you think we ought to include this as part of this story to lay the groundwork for scheduling scripts, or wait to make sure we need it?

Reply:
My gut says we wait until we need it. Up to you.